src/messages/manifestjson.js

```diff
@@ -455,6 +455,17 @@ export const WRONG_ICON_EXTENSION = {
   file: MANIFEST_JSON,
 };
 
+export const IGNORED_APPLICATIONS_PROPERTY = {
+  code: 'IGNORED_APPLICATIONS_PROPERTY',
+  message: i18n._(
+    'Usage of both "browser_specific_settings" and "applications"'
+  ),
+  description: i18n._(
+    oneLine`The applications property is being ignored because it is superseded by the browser_specific_settings property which is also defined in your manifest. Consider removing applications.`
```

Comment:
Nitpick: would you mind to quote both `applications` and `browser_specific_settings` in this message? (I mean like `applications` => `"applications"` and the same for `browser_specific_settings`).